src/main/java/net/javadiscord/javabot/systems/commands/FormatCodeCommand.java

```diff
@@ -19,25 +19,37 @@
  * Command that allows members to format messages.
  */
 public class FormatCodeCommand implements ISlashCommand, IMessageContextCommand {
+	long id;
```

Comment:
Why is `id` a class variable

Reply:
i will make this private, it is a class variable because its being used in lambda expressions and the require variables to be final or atomic, to keep things simple it was better to make it a class variable

Reply:
Just create the `id` variable only in the blpcks you need it (`queue` and `else`)?

Reply:
this wont work as variables are used both in and outside lambda functions (As Far As I Know)